tools/ci/mkchlog-snippet.py

```diff
@@ -89,14 +89,6 @@ def as_snippet(self) -> str:
                 + " via "
             )
         item += f"{self.as_link()} (by {self.author.as_link()})"
-        item = textwrap.fill(
-            item,
-            width=79,
-            initial_indent="- ",
-            subsequent_indent="  ",
-            break_long_words=False,
-            break_on_hyphens=False,
-        )
         return f"### {self.category}\n\n{item}\n"
```

Comment:
```suggestion
        return f"### {self.category}\n\n- {item}\n"
```

`textwrap.fill()` previously added in the hyphen.

Reply:
applied , thanks